src/documents/intern.json

```
@@ -0,0 +1,593 @@
+{
+    "interns": {
+      "Web Development": {
+        "Ali Taquie": {
+            "location": "Watsonville",
+            "department": "Web Development",
+            "schedule": {
+                "Tuesday": "Full Day",
+                "Wednesday": "Full Day",
+                "Thursday": "Full Day",
+                "Friday": "No Available"
```

I think we should change these fields to `Unavailable` instead of `No Available`

```suggestion
                "Friday": "Unavailable"
```